pyClient/test/test_merkle_tree.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2015-2019 Clearmatics Technologies Ltd
+#
+# SPDX-License-Identifier: LGPL-3.0+
+
+
+from zeth.merkle_tree import MerkleTree, PersistentMerkleTree, ZERO_ENTRY
+from os.path import exists, join
+from os import makedirs
+from shutil import rmtree
+from unittest import TestCase
+
+MERKLE_TREE_TEST_DIR = "_merkle_tests"
+MERKLE_TREE_TEST_SIZE = 16
```

Comment:
Let's rename this constant to be clearer. I imagine that by "size" you mean the nb of leaves and not the depth, right?

Reply:
Renamed the constant.